### PR TITLE
Cleanup spacing and other styles of demo

### PR DESCRIPTION
**Pull request by Hirse**
Before 
old

After
new

### Changes
* Rework and align spacing
* Align padding of code blocks with theme padding (and remove schoolbook padding)
* Give lists a bit more width to have fewer line breaks.

### Checklist
- [ ] ~Added markup tests~, or they don't apply here because no markup changes
- [ ] ~Updated the changelog at `CHANGES.md`~ only website changes

### CODE REVIEW

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Can we line up the arrow better? Looks like it's 1-2px off?

**Comment by Hirse** [comment]
> 
> 
> Can we line up the arrow better? Looks like it's 1-2px off?

Does this look better to you now?
image

**Comment by joshgoebel** [comment]
Better. Are you checking all this in Safari also?

**Comment by Hirse** [comment]
> 
> Better. Are you checking all this in Safari also?

No, I don't have a Mac.

File: demo/style.css [hunk 1]
```diff
@@ -122,13 +116,13 @@ aside ul a.current:before {
...
-  margin-left: 18em;
+  margin-left: 19em;
   min-width: 36em;
-  padding: 1px 4em;
+  padding: 1.5em 3em 0;
 }
 
 main > div {
-  margin: 2.5em 0 3em;
+  margin-bottom: 3em;
```
Comment (joshgoebel) [inline demo/style.css:125]: Please revert, the top deserves more margin.

File: demo/style.css [hunk 2]
```diff
@@ -122,13 +116,13 @@ aside ul a.current:before {
 }
 
 main {
-  margin-left: 18em;
+  margin-left: 19em;
   min-width: 36em;
-  padding: 1px 4em;
+  padding: 1.5em 3em 0;
```
Comment (joshgoebel) [inline demo/style.css:121]: Or this is possibly the problem... hard to say.

Comment (joshgoebel) [reply demo/style.css:121]: I don't think the two top titles are equatable in any way so they do not need to "line up" if that is what you were going for.

**Comment by joshgoebel** [comment]
This simply includes "too much" and too intermingled... there seem to be CSS changes other than the ones explicitly documented in the issue text and it's difficult to properly review "looks better to me" PRs like this (hence the delay, but my apologies for that)... often I'll miss something and then just have to revert it later, or someone will break something in Safari because they were only proofing with Chrome etc... I'm also not convinced all of these changes were needed - though it's hard to judge with them all bundled together.

I did try to look at the commit history but it just seems to be a progression rather than "one change per commit", which would make this easier to follow.  If you feel a few of these changes were critical I'd be happy to see a new PR with individual commits (for easy review - to make sure we aren't changing more than intended)... Or also happy to take a second look at this if you wanted to go back and refactor it into a bunch of tiny individual commits detailing each change and the thinking behind it.

One quick example making the example font larger isn't a terrible idea... but that could be decided separately from other layout changes, etc...

**Pull request closed without merging**